### DISCUSSION TITLE
Flatcar LXC image?

### DISCUSSION THREAD

**Discussion by jaskij** [flatcar/Flatcar#963] [General]
I want to deploy Flatcar on top of Proxmox, preferably in an LXC container. Are there any existing images out there, or am I on my own?

**Comment by pothos** [comment]
For containers we have https://stable.release.flatcar-linux.net/amd64-usr/current/flatcar-container.tar.gz but it's not really tested in the release tests.

**Reply by till** [reply]
I think I tried it before. Used that as rootfs and had to create metadata myself. It's been over a year though. But it was working.

**Reply by jaskij** [reply]
Thanks for the info, I'll have a look.
